nsxt/resource_nsxt_compute_manager.go

```diff
@@ -435,7 +435,7 @@ func setCredentialValuesInSchema(d *schema.ResourceData, credential *data.Struct
 		elem["thumbprint"] = credEntry.Thumbprint
 		elem["password"] = credEntry.Password
```

Comment:
Would we see non-empty diff here since sensitive data is not sent back by NSX?

Reply:
Yep - in transport node the polymorphic objects are recreated with make(map[]interface{}), while here the same map is being populated with the GET values. So passwords are retained from the state.
I'm implementing a similar mechanism for transport node.

Reply:
Sorry - I'm still missing something.. I see `elem` is initialized as an empty map, and then populated with `GET` values. I'm missing the part where passwords are coming from the state..